app/controllers/airlines_controller.rb

```diff
@@ -0,0 +1,33 @@
+class AirlinesController < ApplicationController
+
+  def index
+    @airlines = Airline.all
+  end
+
+def show
```

Comment:
Fix indentation here

Reply:
Actually, all these lines should be indented, because they're within 
`class ...`
...
`end`